.goreleaser.yaml

```diff
@@ -10,10 +10,9 @@ builds:
     goarch:
       - amd64
       - arm64
-    ldflags: -w -s -X github.com/kubesimplify/ksctl/selfupdate.BuildDate={{.Date}} -X github.com/kubesimplify/ksctl/selfupdate.GitSummary={{.Version}}
+    ldflags: -w -s -X github.com/kubesimplify/ksctl/cli/cmd.BuildDate={{.Date}} -X github.com/kubesimplify/ksctl/cli.cmd.Commit=$(git rev-parse --short HEAD) -X github.com/kubesimplify/ksctl/cli/cmd.Version=$(git describe --tags --abbrev=0)
```

Comment:
so you have added the Version tag as well ?

Reply:
```bash
-X github.com/kubesimplify/ksctl/cli/cmd.Version=$(git describe --tags --abbrev=0
```
For version, I'm relying on this but AFAIK this one is not working. This was the reason for draft PR.

Reply:
Okay will check and fix this